pkg/web/pages.go

```diff
@@ -79,3 +79,7 @@ func (s *Server) SendEnvelopeForm(c *gin.Context) {
 func (s *Server) TravelAddressUtility(c *gin.Context) {
 	c.HTML(http.StatusOK, "traveladdress.html", scene.New(c))
 }
+
+func (s *Server) UserSettings(c *gin.Context) {
```

Comment:
I'm not thinking this should be UserManagement since there will be an option for individual users to manage their password and potentially other settings in the future.

Reply:
Perhaps we could call this `UserManagement` and call the option for individual users to manage their password `ProfileManagement`?